apps/dsp-app/src/app/workspace/resource/representation/representation.service.ts

```diff
@@ -78,6 +57,20 @@ export class RepresentationService {
         );
     }
 
+    /**
+     * return the jwt token from the session to authenticate
+     * @return the token
+     */
+    public getTokenFromLocalStorage(): string {
```

Comment:
This is the same method that you have below but is private. Should it be public or private?

Reply:
Damn, was due to merging. (I changed it to public in order the pdf viewer could possibly also send the token.)